pwiz_tools/Skyline/Properties/Resources.resx

```diff
@@ -10803,4 +10803,8 @@ There might be something wrong with default web browser on this computer.</value
   <data name="CommandLine_AddDecoys_Decoys_discarded" xml:space="preserve">
     <value>Decoys discarded</value>
   </data>
+  <data name="CachedLibrary_WarnInvalidEntries_" xml:space="preserve">
+    <value>{0} of the {1} entries in library '{2}' were invalid, including:
```

Comment:
Not a big fan of messages starting with a number. How about this?

Invalid entries found in the library '{0}' ({1} of {2}), including:

This gets to the problem statement quicker with the numbers being more supplementary information, which they are.